source/server/spongineer/commands.rst

```diff
@@ -60,6 +60,19 @@ Command                 Description                               Permission
 * /sponge save [-g] [-d dim|*] [-w world|*]
 * /sponge reload [-g] [-d dim|*] [-w world|*]
 
+**Command Conflicts**
+
+In cases of command conflict, Sponge provides a primary alias mechanism to specify which command is to be used.  For 
+example, Minecraft provides the `reload <https://minecraft.gamepedia.com/Commands#Summary_of_commands>`__ command and 
+Sponge provides the `reload <https://docs.spongepowered.org/stable/en/server/spongineer/commands.html>`__ command. To 
+specify which command to use, prefix it with minecraft or sponge and a ``:``. So, to use Sponge's reload command 
+above, type in ``/sponge:reload``. This approach can also be used to handle conflicts between mods and/or plugins. Do the 
+same thing, just use the mod-id or the plugin-id and a ``:``.  An example is ``/nucleus:tp``.
```

Comment:
Should we make a mention of specific plugins in the docs? Just a thought, otherwise everything else looks fine.

Reply:
We could use `/examplemod:reload` as an alternative. But IMO that does not matter.